src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -83,4 +84,14 @@ public static void main(String[] args) {
             return;
         }
     }
+
+    public  void createAccount() {
+        System.out.println("Whats your name?");
+        ui.createScanner();
```

Comment:
If you create a scanner, please try to close the scanner in the same function. Not that it matters, since it just overwrites ui.scanner, but I don't the program doing anything that is not explicitly coded out. Makes debugging just a little bit harder

Reply:
Erm wait, cancel the previous comment, I did not realise createScanner() was already called before createAccount was called. Since createScanner() is already called, you don't need to call it again, you can just use ui.nextLine() straightaway